app/views/instances/show.html.erb

```diff
@@ -0,0 +1,5 @@
+<% game = Game.find(@instance.game_id)%>
+<% player = User.find@instance.user_id%>
```

Comment:
Thou shall not query from views to prevent performance issues.

Lets move these to have `@game` and `@player` from `app/controllers/instances_controller.rb` please 🙏🏼 .

Reply:
Fixed!